File: src/main/java/com/learning_platform/learning_resources/dto/QuestionCredentials.java
package com.learning_platform.learning_resources.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class QuestionCredentials {
    private String question;
    private String connectedUser;
    private String date;

}

File: app/model/QuestionCredentials.ts
export class QuestionCredentials{
  question:string;
  connectedUser:string;
  date:string;


  constructor(question:string,connectedUser:string,date:string) {
    this.question=question;
    this.connectedUser=connectedUser;
    this.date=date;
  }

}
